draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1796,13 +1798,13 @@ if D > 1:
     for v in Split(tags, 128):
         Absorb(ZeroPad(v, R))
 
-    u = LE64(D) || LE64(tag_length_bits)
+    u = LE64(D) || LE64(tag_len_bits)
     t = ZeroPad(V[3,0] ^ u, R)
     Repeat(7, Update(t))
 
-if tag_length_bits == 128:
+if tag_len_bits == 128:
     tag = V[0,0] ^ V[1,0] ^ V[2,0] ^ V[3,0] ^ V[4,0] ^ V[5,0] ^ V[6,0]
```

Comment:
Doesn't this correspond to AEGIS-128X, not AEGIS-256X? AEGIS-256X doesn't have this many states.

Reply:
Oops, good catch, thanks!